plugins/Cargo.toml

```diff
@@ -9,12 +9,12 @@ repository = "https://github.com/centricular/webrtcsink/"
 build = "build.rs"
 
 [dependencies]
-gst = { package = "gstreamer", version = "0.18", features = ["v1_20", "ser_de"] }
-gst-app = { package = "gstreamer-app", version = "0.18", features = ["v1_20"] }
-gst-video = { package = "gstreamer-video", version = "0.18", features = ["v1_20", "ser_de"] }
-gst-webrtc = { package = "gstreamer-webrtc", version = "0.18", features = ["v1_20"] }
-gst-sdp = { package = "gstreamer-sdp", version = "0.18", features = ["v1_20"] }
-gst-rtp = { package = "gstreamer-rtp", version = "0.18", features = ["v1_20"] }
+gst = { git="https://gitlab.freedesktop.org/gstreamer/gstreamer-rs", rev="61a3f5296c3093e9811d883256ad38bd5483b6db", package = "gstreamer", features = ["v1_20", "ser_de"] }
```

Comment:
@sdroege does that seem OK to you? :)

Reply:
We depend on `gstreamer-rs` main, but decided to pin a specific commit here.

Reply:
Well if you want to keep this up to date :shrug:

Reply:
ok thanks, I think given webrtcsink should eventually live in `gst-plugins-rs`, we probably should do what's done there and just depend on main.

Reply:
You probably want to update after https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/-/merge_requests/1019 is merged

Reply:
@vivienne-w let's just use main :)

Reply:
(ie you can remove the `rev` altogether)

Reply:
We still want to pin in the `.lock` file I believe.